global_translations.R

```diff
@@ -38,3 +38,27 @@ complete_trans_df <- subset(final_df, untranslated_count == 0, select = -c(fuzzy
 
 # Untranslated Packages with respective languages associated
 complete_untrans_df <- subset(final_df, translated_count == 0, select = -c(fuzzy_count))
+
+# R Language Translation Team Contact details
+library(rvest)
+library(stringr)
+
+translation_teams <- read_html("https://developer.r-project.org/TranslationTeams.html") |>
+  html_node("table") |>
+  html_table()
+
+members_contact <- translation_teams$Contact |>
+  gsub(pattern = ".*<", replacement = "") |>
+  gsub(pattern = ">", replacement = "") |>
+  gsub(pattern = ",.*", replacement = "") |> 
+  str_trim(side = "right")
+
+lang_members <- translation_teams$Contact |>
```

Comment:
ditto here -- any reason not to assign to a column directly? maybe this is a leftover from the previous setup where we parsed the page then made a table. but now we start with a table and edit it.

Reply:
No specific reason. Was going with the follow. Will directly assign these to data frame.
Thanks